src/test/kotlin/com/ampnet/auditornode/Assertions.kt

```diff
@@ -26,3 +27,39 @@ fun <A, B> Assert<Either<A, B>>.isLeftContaining(expected: A) =
 
 fun <A, B> Assert<Either<A, B>>.isRightContaining(expected: B) =
     isRightSatisfying { assertThat(it).isEqualTo(expected) }
+
+@Language("JavaScript")
+val jsAssertions = """
+    function assertEquals(name, expected, actual) {
+        if (expected !== actual) {
+            let message = "Assertion failed for '" + name + "'; expected: '" + expected + "', actual: '" + actual + "'";
```

Comment:
Can we use `const` instead of `let`?

Reply:
As discussed offline, using `const` instead of `let` does not make much difference (in tests)... Also this line fits nicely into the 120 column limit with `let`, so I will leave it as is 😄